### PR TITLE
Fix Golint errors

### PR DESCRIPTION
**Pull request by gdbelvin**
- Add comments to public functions.
- Use camel case for variable names.
- Export types that are returned by public functions. (`GetEntriesResponse`)

### CODE REVIEW

**Comment by gdbelvin** [comment]
@taknira @pphaneuf

File: go/client/getentries.go [hunk 1]
```diff
@@ -13,20 +13,19 @@ import (
...
+// LeafEntry respresents a JSON leaf entry.
 type LeafEntry struct {
 	LeafInput []byte `json:"leaf_input"`
 	ExtraData []byte `json:"extra_data"`
 }
 
-// getEntriesReponse respresents the JSON response to the CT get-entries method
-type getEntriesResponse struct {
+// GetEntriesResponse respresents the JSON response to the CT get-entries method.
+type GetEntriesResponse struct {
```
Comment (pphaneuf) [inline go/client/getentries.go]: I guess I was lucky with that, my only use was of a `entries, err := GetRawEntries(...)` form, so it worked anyway, even if I couldn't explicitly declare that `entries` variable!

**Comment by pphaneuf** [comment]
There's a build error, you need to update the mention of `getEntriesResponse` at line 46 of `getentries.go`.

**Comment by gdbelvin** [comment]
Fixed errors

**Comment by pphaneuf** [comment]
LGTM, I'll merge this shortly (waiting on the last Travis build, just to be safe!).

**Pull request merged**